### PR TITLE
Add Google Maps Service and moving Google Maps Key to env file

### PR DESCRIPTION
**Pull request by nathanqueija**

### CODE REVIEW

File: env-config.js [hunk 1]
```diff
@@ -12,11 +12,13 @@ if (!prod) {
...
-  'process.env.REACT_APP_API_URL': process.env.WEBSERVICE_BASE_URL || 'http://localhost:4000',
+  'process.env.REACT_APP_API_URL':
+    process.env.WEBSERVICE_BASE_URL || 'http://localhost:4000',
   'process.env.REACT_APP_CLOUDINARY_BASE_URL':
     'https://res.cloudinary.com/emcasa/image/upload',
   'process.env.REACT_APP_FAVICON': favicon,
   'process.env.GOOGLE_ANALYTICS_TRACKING_ID': prod
     ? process.env.GOOGLE_ANALYTICS_TRACKING_ID
     : null,
+  'process.env.GOOGLE_MAPS_KEY': process.env.GOOGLE_MAPS_KEY || null
```
Comment (gusaiani) [inline env-config.js]: @nathanqueija, neste momento, vc está iniciando a key quando roda o app no terminal?

Não é melhor colocar aqui direto tipo:

```
'process.env.GOOGLE_MAPS_KEY`: process.env.GOOGLE_MAPS_KEY || $umaKeyReal
```

Comment (nathanqueija) [reply env-config.js]: I considered this option, but thought it would not be a good idea to leave the key publicly exposed.

Comment (gusaiani) [reply env-config.js]: How would the workflow be locally?

Comment (gusaiani) [reply env-config.js]: The key is publicly exposed in the js itself, in the client. So perhaps it doesn't matter.

Comment (nathanqueija) [reply env-config.js]: I'm starting the node process and setting the variable.
For example:
`GOOGLE_MAPS_KEY = 1234example456 yarn dev`
You're right. There is another file in which the key is already publicly exposed.
If I change there to pick up the process variable do you think it's worth hiding the key and always starting the process with it?

Comment (nathanqueija) [reply env-config.js]: Ok, I'll do that for now

**Pull request merged**